spec/controllers/expenses_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
```

Comment:
Unit and Integration specs are well-written and sufficient covering all controllers, views, and models 🎆 🍾 

![image](https://user-images.githubusercontent.com/38088818/236582541-a0a36a7a-8828-4cea-9d41-726fa6c353dc.png)

Well done! 💯